### ISSUE TITLE
Heapsnapshot format changed!

### ISSUE THREAD

**Issue by ForesightImaging** [SrTobi/v8-heapsnapshot#2]
I've just tried out this package in an effort to add some tracking to our ci server. I can now create parse a snapshot after key e2e tests to be sure that certain elements aren't leaked. It's a fantastic tool to prevent regressions.

However, I'm seeing a message on the console: 
**Heapsnapshot format changed! Please report to https://github.com/SrTobi/v8-heapsnapshot/issues**

So I'm doing it. Thanks!

Our runtime environment is electron 5, Chromium to 73
The line that is causing the trouble is 49:
`check(JSON.stringify(meta) === JSON.stringify(metaData));
`
where:
```
"{
    "node_fields": [
        "type",
        "name",
        "id",
        "self_size",
        "edge_count",
        "trace_node_id"
    ],
    "node_types": [
        [
            "hidden",
            "array",
            "string",
            "object",
            "code",
            "closure",
            "regexp",
            "number",
            "native",
            "synthetic",
            "concatenated string",
            "sliced string",
            "symbol",
            "bigint"
        ],
        "string",
        "number",
        "number",
        "number",
        "number",
        "number"
    ],
    "edge_fields": [
        "type",
        "name_or_index",
        "to_node"
    ],
    "edge_types": [
        [
            "context",
            "element",
            "property",
            "internal",
            "hidden",
            "shortcut",
            "weak"
        ],
        "string_or_number",
        "node"
    ],
    "trace_function_info_fields": [
        "function_id",
        "name",
        "script_name",
        "script_id",
        "line",
        "column"
    ],
    "trace_node_fields": [
        "id",
        "function_info_index",
        "count",
        "size",
        "children"
    ],
    "sample_fields": [
        "timestamp_us",
        "last_assigned_id"
    ],
    "location_fields": [
        "object_index",
        "script_id",
        "line",
        "column"
    ]
}"
```

and JSON.stringify(metaData,null,4)

```
"{
    "node_fields": [
        "type",
        "name",
        "id",
        "self_size",
        "edge_count",
        "trace_node_id"
    ],
    "node_types": [
        [
            "hidden",
            "array",
            "string",
            "object",
            "code",
            "closure",
            "regexp",
            "number",
            "native",
            "synthetic",
            "concatenated string",
            "sliced string",
            "symbol"
        ],
        "string",
        "number",
        "number",
        "number",
        "number",
        "number"
    ],
    "edge_fields": [
        "type",
        "name_or_index",
        "to_node"
    ],
    "edge_types": [
        [
            "context",
            "element",
            "property",
            "internal",
            "hidden",
            "shortcut",
            "weak"
        ],
        "string_or_number",
        "node"
    ],
    "trace_function_info_fields": [
        "function_id",
        "name",
        "script_name",
        "script_id",
        "line",
        "column"
    ],
    "trace_node_fields": [
        "id",
        "function_info_index",
        "count",
        "size",
        "children"
    ],
    "sample_fields": [
        "timestamp_us",
        "last_assigned_id"
    ]
}"
```

The diffs yield that there's a new node_type, 'bigint' and there's a new section:
```
    "location_fields": [
        "object_index",
        "script_id",
        "line",
        "column"
    ]

```

**Comment by SrTobi** [comment]
Thanks @ForesightImaging for reporting. I added the new 'bigint' type and the location_fields to the sanity check. And I added bigint to the NodeTypes in the public api. I didn't do anything else with the location_fields. It can be added if someone actually needs it

Changes are published as version `1.1.0`.

**Comment by ForesightImaging** [comment]
thanks!

I don't currently have a use for those fields, if the automated tests detect a leak regression humans will have to track down the trouble.
